lua/neogit/status.lua

```diff
@@ -5,7 +5,9 @@ local cli = require('neogit.lib.git.cli')
 local notif = require("neogit.lib.notification")
 local config = require("neogit.config")
 local a = require 'plenary.async_lib'
-local async, await, await_all, future, void, scheduler, run = a.async, a.await, a.await_all, a.future, a.void, a.scheduler, a.run
+-- luacheck: ignore run future
```

Comment:
same thing here

Reply:
Removed everything unused now. I originally thought that you may had a reason to always import this set of functions in the same manner in all files but they can just be imported when needed.